source/beamline/IDEAS/IDEASKETEKDetailedDetectorView.cpp

```diff
@@ -41,7 +38,7 @@ void IDEASKETEKDetailedDetectorView::onPeakingTimeBoxChanged(const QString &arg1
 	}
 	else if (arg1 == "High Res / Low Rate")
 	{
-		ketekDetector_->setPeakingTime(0.200);
+		ketekDetector_->setPeakingTime(2.00);
```

Comment:
Correct.  @epengr made a comment about it in one of the issues.  I don't remember which one.